ESLint: custom directories from next.config.js (#26401)
vercel/next.js: docs/basic-features/eslint.md

## Linting Custom Directories

By default, Next.js will run ESLint for all files in the `pages/`, `components/`, and `lib/` directories. However, you can specify which directories using the `dirs` option in the `eslint` config in `next.config.js`:

```js
module.exports = {
  eslint: {
    dirs: ['pages', 'utils'], // Only run ESLint on the 'pages' and 'utils' directories (next build and next lint)
  },
}
```

Also, the `--dir` flag can be used for `next lint`:

```bash
yarn lint --dir pages --dir utils
```

> and it will take precedence over the configuration from `next.config.js`.

Reviewer: Tiny nit :)

```suggestion
This will take precedence over the configuration from `next.config.js`.
```
